mariel/day-01/day-01-part2.js

```diff
@@ -0,0 +1,26 @@
+import fs from "fs";
+import path from "path";
+
+const __dirname = path.resolve();
+
+const day01part2 = () => {
+  const input = fs
+    .readFileSync(path.join(__dirname, "day-01-input.txt"), "utf-8")
+    .split("\n")
+    .map(Number);
+
+  let count = 0;
+
+  for (let i = 0; i < input.length; i++) {
+    if (
+      input.slice(i, i + 3).reduce((a, b) => a + b) <
+      input.slice(i + 1, i + 4).reduce((a, b) => a + b, 0)
```

Comment:
Any reason one of these reductions specifies an initial value and the other doesn't? What happens when you slice past the end of the list (i.e. when `i` is within 3 of the total input length)?